event/events.go

```diff
@@ -47,12 +54,21 @@ func Multiplex(events ...EventEmitter) *multiplexedEvents {
 
 // The events struct has methods for working with events.
 type events struct {
-	eventSwitch *evts.EventSwitch
+	eventSwitch *go_events.EventSwitch
 }
 
 // Subscribe to an event.
-func (this *events) Subscribe(subId, event string, callback func(evts.EventData)) error {
-	this.eventSwitch.AddListenerForEvent(subId, event, callback)
+func (this *events) Subscribe(subId, event string,
+	callback func(txs.EventData)) error {
+	cb := func(evt go_events.EventData) {
+		eventData, err := mapToOurEventData(evt)
+		if err != nil {
+			panic(fmt.Sprintf("Failed to map go-events EventData to our EventData %v",
```

Comment:
So I think this is addressed in previous comment. I don't have a nice option here for returning an error along the callback chain.

I am leaning towards `log.Error`ing